buddyboss/buddyboss-platform
# Non-admin have HTML tags showing on forum replies

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #867
Fixes #1226

### How to test the changes in this Pull Request:

1. As a Subscriber, go to Discussions and reply to one
2. Copy your content from a web page
3. Paste the content and reply this in the discussion
4. See HTML Tags are showing or not.

### Proof Screenshots or Video
http://somup.com/cY1oiqQYiS


### Other information:
